templates/download/index.html

```diff
@@ -2,208 +2,227 @@
 
 {% block title %}Get Ubuntu | Download{% endblock %}
 
-{% block meta_description %}Download Ubuntu desktop, Ubuntu Server, Ubuntu for Raspberry Pi and IoT devices, Ubuntu Core
-and all the Ubuntu flavours. Ubuntu is an open-source software platform that runs everywhere from the PC to the server
-and the cloud.{% endblock %}
+{% block meta_description %}
+  Download Ubuntu desktop, Ubuntu Server, Ubuntu for Raspberry Pi and IoT devices, Ubuntu Core
+  and all the Ubuntu flavours. Ubuntu is an open-source software platform that runs everywhere from the PC to the server
+  and the cloud.
+{% endblock %}
 
-{% block meta_copydoc %}https://docs.google.com/document/d/1egsCcmFiz8czeHsvwoB9Jz815Ij5Urtf7Ngrru6Zd4o/edit{% endblock
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1egsCcmFiz8czeHsvwoB9Jz815Ij5Urtf7Ngrru6Zd4o/edit
+{% endblock
 meta_copydoc %}
 
-{% block meta_image %}https://assets.ubuntu.com/v1/4d42e36c-Jammy+Jellyfish+RGB.svg{% endblock %}
+{% block meta_image %}https://assets.ubuntu.com/v1/c5414d0e-Numbat.png{% endblock %}
 
 {% block body_class %}is-paper{% endblock %}
 
 {% block content %}
 
-<section class="p-strip is-shallow u-no-padding--bottom">
-  <div class="row--50-50 p-section">
-    <div class="col">
-      <h1>Download Ubuntu</h1>
-    </div>
-    <div class="col">
-      <div class="p-section--shallow">
-        <p>Ubuntu is the world's favourite Linux operating system. Run it on your laptop, workstation, server or IoT device, with five years of free security updates.</p>
+  <section class="p-strip is-shallow u-no-padding--bottom">
+    <div class="row--50-50 p-section">
+      <div class="col">
+        <h1>Download Ubuntu</h1>
       </div>
-      <div class="p-image__background--3-2 u-vertically-center">
-        {{ 
-          image (
-            url="https://assets.ubuntu.com/v1/69e634af-laptop-forubuntu-desktop.png",
-            alt="",
-            width="1801",
-            height="954",
-            hi_def=True,
-            loading="auto"
-            ) | safe
-        }}
+      <div class="col">
+        <div class="p-section--shallow">
+          <p>
+            Ubuntu is the world's favourite Linux operating system. Run it on your laptop, workstation, server or IoT device, with five years of free security updates.
+          </p>
+        </div>
+        <div class="u-vertically-center">
+          {{ image(url="https://assets.ubuntu.com/v1/622b4fe1-numpy-numbat-on-dell-xps-13.jpg",
+                    alt="",
+                    width="1801",
+                    height="954",
+                    hi_def=True,
+                    loading="auto") | safe
```

Comment:
```suggestion
                    loading="auto" | safe
```

Reply:
This is actualy correct, it opens at `image(` and closes `..."auto")|safe` it's just the formatting of djlint that makes it confusing...